integration/agent/agent_test.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Ant Group
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This will run a containers and then check
+# whether the kata-agent threads namespaces
+# were changed during create container processes.
+
+set -e
```

Comment:
Why not add all the "boiler-plate" code here?

```bash
set -o errexit
set -o nounset
set -o pipefail

[ -n "$BASH_VERSION" ] && set -o errtrace
[ -n "${DEBUG:-}" ] && set -o xtrace
```

Alternatively, this could be a `bats` test though.